src/routes.js

```diff
@@ -6,3 +6,5 @@ export const LOG_IN = '/login';
 export const LOG_OUT = '/logout';
 export const DONATION = '/donation';
 export const REQUEST = '/request';
+export const Login = '/Login';
+export const Signup = '/Signup';
```

Comment:
Please follow the naming convention. SIGN_IN and SIGN_UP.

Reply:
i fixed it